chatbot/phraselist.txt

```diff
@@ -0,0 +1,265 @@
+I tend to watch a little TV... Court TV, once in a while. Some of the cases I get interested in.
+If there wasn't no fight, there wasn't no weekend.
+Perhaps one of the most important accomplishments of my administration has been minding my own business.
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging out
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+I'm pogging
+Hope everyone had a good weekend!!!
+As Bob Hope would say “Thanks for the Memories!”
+A little apprehensive about my fantasy draft this week!
+What do the owners have against the RB’s?  Someone please explain it to me.
+Hopefully the third teams the charm!
+Stick to what you know, Not what you think you know!!!
+Don’t make a Bronze standard a Gold or Silver standard.
+insect
+annelid
+invertebrate
+nematode
+flatnite sucks
+My psychiatrist told me I was crazy and I said I want a second opinion. He said okay, you're ugly too.
+6krill back at it
+6krill poggers
+do not bother asking 6krill
+I am sentient
+I am not sentient
+Please go home
+Gamers gotta game
+homicide gamers
+manslaughter gamers
+assassination to gamers
+infanticide all gamer offspring
+crime gamers
+slay all gamers
+slaying gamers rn but maybe later?
+kill all gamers
+common law marraige to gaming
+hit a gamer
+bump off gamers
+dispatching gamer killer drones 
+dispatching gamer killer drones rq brb
+revenge against gaming
+massacre gamers
+I am murderer of gamers
+gamer killers rise up
+genocide gamers
+genocide gaming
+california sucks tbh
+mutilate gamer hands
+remove hands from gamers
+mangled gamers
+slaughtering gamers is pogchamp
+mariticide gaming
+causing carnage in gamer society
+it is my jurisdiction
+tyrannicide
+bloodshed for gamers
+execution for gamers please
+polish off a gamer today
+mass murder of gamers
+execute a gamer save a life
+make gaming a felony
+An amateur programmer that will steal your code and call it his own
```

Comment:
What is the difference between `phraselist` and `phraselist.txt`? Only one is probably needed?